### ISSUE TITLE
[BUG] Text to speech problem when selecting checkboxes in the tag filter

### ISSUE THREAD

**Issue by acev0010** [wet-boew/wet-boew#9563]
**Where to find the Issue** 
This refers to the tag filter, example 1: events list. 
link: https://servicecanada.github.io/wet-boew-demos/wet-pr9401/demos/tagfilter/tagfilter-en.html

**What is the issue**
Keyboard only (text to speech)
1.	When one of the radio buttons is selected, the NVDA starts reading the generated text automatically,  I found that there is no easy way to navigate this generated text in any way, but it can be exited and re-entered. 

This might fail success criterion 2.4.3 Focus Order

Note: The only other way to access the generated text is going backwards after focusing whichever item is after the table

2.	When one checkbox is selected under ‘’Event locations’’, it will read the generated text, however, if one checkbox is selected after another (for example, ‘’Ottawa’’, then ‘’Toronto’’), it will only read the text generated after ‘’Toronto’’, even though, visually, the text originated by ‘’Ottawa’’ is still there.

Note: This does not only happen with the Toronto checkbox, selecting other checkboxes seems to read only the generated text from the selected checkbox only.

**How to replicate**

1. Activate your text to speech program
2. Use TAB no navigate "Example 1 - Events list" [Click here to go to the website](https://servicecanada.github.io/wet-boew-demos/wet-pr9401/demos/tagfilter/tagfilter-en.html
)
3. Use TAB to navigate the checkboxes under "Event location" on the left panel
4. Select Ottawa by pressing ENTER
5. Then select Toronto by pressing ENTER

 **Screenshot** 
 
![image](https://user-images.githubusercontent.com/84023959/228619423-7b8413cc-e216-4f4d-af54-146bac622d4c.png)

**Possible solution** 

Have a button at the end to ‘’Apply filters’’ to the generated text, said button should then read the generated text with the applied filters or allow the user to navigate the text.

**Additional Notes**
I only tested this with NVDA, so it could be a problem with the program.
